storage/src/backend/connection.rs

```diff
@@ -203,9 +236,32 @@ impl Connection {
         } else {
             None
         };
+
+        let mut mirrors = Vec::new();
+        for mirror_config in config.mirrors.iter() {
+            if !mirror_config.host.is_empty() {
+                mirrors.push(Arc::new(Mirror {
+                    // TODO: build mirror connection from MirrorConfig, now is from registry config
+                    client: Self::build_connection("", config)?,
```

Comment:
it is unnecessary to build the client on initializing.

Reply:
If every request built client to mirror, does it affect performance? 
Or do you mean to reuse the `state.client`? just like:
```
self.call_inner(
            &self.client,
...
```

Reply:
Yes, should reuse the same client.

Reply:
Got it.